pandas_gbq/gbq.py

```diff
@@ -406,6 +416,20 @@ def to_gbq(
     destination_table : str
         Name of table to be written, in the form ``dataset.tablename`` or
         ``project.dataset.tablename``.
+    clustering_columns : typing.Union[
+        pandas.core.indexes.base.Index, typing.Iterable[typing.Hashable]
+    ] = (),
```

Comment:
Getting a docs failure. I think it might be this line.

```
sphinx.errors.SphinxWarning: /tmpfs/src/github/python-bigquery-pandas/pandas_gbq/gbq.py:docstring of pandas_gbq.gbq.to_gbq:19:Field list ends without a blank line; unexpected unindent.
```

Let's try this.

```suggestion
    clustering_columns : pandas.Index | Iterable[Hashable]
```